src/debugging/python/PythonDebugHelper.ts

```diff
@@ -36,10 +35,6 @@ export interface PythonDockerDebugConfiguration extends DockerDebugConfiguration
 }
 
 export class PythonDebugHelper implements DebugHelper {
```

Comment:
Is there any issue from bringing existing debug config / tasks (old scaffolding) into this new model?

Reply:
Not that I can think of, we previously had port and volume mappings specific for the debugger but now are not needed. The only compatibility issue I can think of is related to the version of the Python extension, so I do an explicit check there.